start.py

```diff
@@ -15,6 +15,7 @@
 start_date = end_date - timedelta(days=days_delta - 1)
 print("Metrics from " ,start_date-timedelta(days=1), "is used to compute recommendation")
 tolerance=int(os.environ.get('TOLERANCE', '0'))
+batch_size=int(os.environ.get('BATCH_SIZE', '1'))
```

Comment:
We should add it to the recommend.sh, right @xrajesh ?

Reply:
As I think more, I think what you have now is fine. Unless you have second thoughts, we can resolve this